recipes/default.rb

```diff
@@ -6,8 +6,19 @@
 #
 node.default['nssm']['install_location'] = '%WINDIR%'
 
+consul_home = node['consul']['service_user_home']
```

Comment:
I see that we use this variable only once. May be, we can omit that and just use an attribute as is?